app/models/manageiq/providers/redhat/infra_manager/vm/reconfigure.rb

```diff
@@ -39,4 +40,103 @@ def spec_for_added_disks(disks)
       :storage => storage
     }
   end
+
+  def spec_for_network_adapters(options)
+    {
+      :edit   => (network_adapters_edit(options[:network_adapter_edit]) if options[:network_adapter_edit]),
+      :add    => (network_adapters_add(options[:network_adapter_add]) if options[:network_adapter_add]),
+      :remove => (network_adapters_remove(options[:network_adapter_remove]) if options[:network_adapter_remove])
+    }.compact
+  end
+
+  def network_adapters_add(adapters)
+    nic_names = hardware.nics.pluck(:device_name)
+    switch_ids = HostSwitch.where(:host_id => host.id).pluck(:switch_id)
```

Comment:
This can be just `.where(:host => host)` no need to have it be `host.id`

Reply:
Yeah right, sorry. I'll fix immediately